lib/twirp/protoc_plugin/code_generator.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
-require_relative "../../google/protobuf/compiler/plugin_pb"
-require_relative "../../core_ext/string/snake_case"
-require_relative "descriptor_ext/service_descriptor_proto_ext"
+require "core_ext/string/snake_case"
+require "google/protobuf/compiler/plugin_pb"
 require "stringio"
+require "twirp/protoc_plugin/descriptor_ext/service_descriptor_proto_ext"
```

Comment:
Personally, I think external requires are neater done first, YMMV
```suggestion
require "stringio"
require "core_ext/string/snake_case"
require "google/protobuf/compiler/plugin_pb"
require "twirp/protoc_plugin/descriptor_ext/service_descriptor_proto_ext"
```